client/lcd/root.go

```diff
@@ -36,7 +36,7 @@ type RestServer struct {
 // NewRestServer creates a new rest server instance
 func NewRestServer(cdc *codec.Codec) *RestServer {
 	r := mux.NewRouter()
-	cliCtx := context.NewCLIContext().WithCodec(cdc)
+	cliCtx := client.NewContext().WithCodec(cdc)
```

Comment:
I think we should rename `cliCtx` -> `ctx`

Reply:
or `cliCtx` -> `clientCtx`

Reply:
I think clientCtx will be better over ctx, will change accordingly